### DISCUSSION TITLE
Hi!

### DISCUSSION THREAD

**Discussion by hello-smile6** [SpellCheckJS/SpellCheckJS#4] [Off-topic maintainer]
@Scoldercreations I noticed you found this! How did you know this existed?
### I checked https://github.com/SpellCheckJS/SpellCheckJS/stargazers

**Comment by hello-smile6** [comment]
@ScolderCreations Hi???

**Reply by ScolderCreations** [reply]
I saw this on my GH homepage.

**Reply by hello-smile6** [reply]
> I saw this on my GH homepage.

Oh! Could you help me with https://github.com/SpellCheckJS/html-client ?

**Reply by hello-smile6** [reply]
@ScolderCreations Are you going to accept the invite?

**Comment by hello-smile6** [comment]
Currently, only I'm watching this repo. :(
https://github.com/SpellCheckJS/SpellCheckJS/watchers
